modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker_{context}"]
-= Creating a broker manually
+= Creating a Broker as a developer
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+. Optional. If you do not have an existing event sink created to consume events from the Broker, you can create a simple Knative Service as an event sink.
+.. Copy the following into a YAML file:
```

Comment:
You need a noun after "following." I'd say something like "Save the following YAML in a `.yaml` file: